src/commands/OSRS_Fun/kill.ts

```diff
@@ -13,7 +13,7 @@ export default class extends BotCommand {
 			cooldown: 1,
 			oneAtTime: true,
 			description: 'Simulate killing OSRS monsters and shows the loot.',
-			usage: '<quantity:int{1}> <BossName:...str>',
+			usage: '<quantity:int{1}> <BossName:...str> [inCatacombs:boolean]',
```

Comment:
Why are you specifying that the usage accepts a bool for inCatacombs if you're manually setting it later exclusively by the value of msg.flagArgs.cannon?

Please see my additional comments as well for a suggested rework.